source/MaterialXRuntime/RtFileIo.cpp

```diff
@@ -866,10 +872,13 @@ namespace
             for (InputPtr input : surfaceShader->getActiveInputs())
             {
                 BindInputPtr bindInput = shaderRef->addBindInput(input->getName(), input->getType());
-                if (input->hasNodeGraphName() && input->hasOutputString() && doc->getNodeGraph(input->getNodeGraphName()))
+                if (input->hasNodeGraphName() && doc->getNodeGraph(input->getNodeGraphName()))
```

Comment:
Nodegraph names shouldn't be paired with output strings if there is a single output, so the logic here has been changed.